GherkinSyncTool/Synchronizers/TestRailSynchronizer/Content/SectionSynchronizer.cs

```diff
@@ -46,18 +45,25 @@ public ulong GetOrCreateSectionIdFromPath(string relativePath)
             return GetOrCreateSectionIdRecursively(_testRailSections, sourceSections, suiteId, projectId);
         }
 
-        public void MoveNotExistingSectionsToArchive(List<IFeatureFile> featureFiles)
+        public void MoveNotExistingSectionsToArchive()
         {
-            var featureFileFolders = GetFeatureFileFolderTree(featureFiles);
+            _archiveSection = _testRailSections.Find(section => section.Name.Equals(_config.TestRailSettings.ArchiveSectionName, StringComparison.InvariantCultureIgnoreCase))?.Id;
```

Comment:
Maybe we should have separate field for **_config.TestRailSettings** since we're addressing it multiple times in the different methods of this class?

Reply:
I think it is ok for now, but it will be a good idea to thinks about more deep refactoring in the future.